lib/model.js

```diff
@@ -1647,6 +1647,7 @@ class Model {
    * @param  {Object}                                                    [options.having] Having options
    * @param  {string}                                                    [options.searchPath=DEFAULT] An optional parameter to specify the schema search_path (Postgres only)
    * @param  {boolean|Error}                                             [options.rejectOnEmpty=false] Throws an error when no records found
+   * @param  {boolean}                                                   [options.subQuery] If true, uses sub-query for querying associates.
```

Comment:
This option should not be used, instead you should be using `duplicating: false` etc (Which are not documented either :D)

Reply:
@sushantdhiman then I propose the following: document both of these features, but mention that subQuery should not be used.
Can you do that? (I'm not that familiar with internals of Sequelize unfortunately and don't know how either of this options works and why one should use duplicating instead of subQuery)

Reply:
Please also add it to the typings.